Alicerce/Sources/Logging/Providers/FileProvider.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  FileProvider.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 03/04/17.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+final class FileProvider {
+    
+    internal var minLevel: Log.Level = .error
+    internal var formatter: LogItemFormatter = LogItemStringFormatter()
+    
+    let fileURL: URL
+    let fileManager = FileManager.default
+    
+    //MARK:- Lifecycle
+    
+    init(fileURL: URL) {
+        self.fileURL = fileURL
+    }
+    
+    //MARK:- Public Methods 
+    
+    public func clear() {
```

Comment:
I'd say it depends; if I see it from a programmer's point of view I'd say the primary concern is to have a cleaned up logfile, and not to write boiler code to check if the log file exists or error handle. It's a logging feature, so my principal concern is logging - if it would be a persistency feature my opinion would be clearly different.